### PR TITLE
Feature/telegram

### PR DESCRIPTION
**Pull request by tylim88**
done telegram send message

remove prettier ignore

### CODE REVIEW

File: packages/components/credentials/Telegram/TelegramApi.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+    constructor() {
+        this.name = 'telegramApi';
+        this.version = 1.0;
+        this.credentials = [
+            {
+                label: 'Bot Token',
+                name: 'botToken',
+                type: 'string',
+                default: 'eg: 1234567890:ABCDEFGHIJKLMNOPQRSTUVWXYZABCDEFGHI',
```
Comment (HenryHengZJ) [inline packages/components/credentials/Telegram/TelegramApi.ts:16]: would be better to leave the default as empty and have this set to `placeholder`

File: packages/components/nodes/Telegram/Telegram.ts [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+                ],
+                default: 'eg: 1234567890:ABCDEFGHIJKLMNOPQRSTUVWXYZABCDEFGHI'
+            }
+        ] as INodeParams[];
+        this.inputParameters = [
+            {
+                label: 'Channel ID',
+                name: 'channelID',
+                type: 'string',
+                default: 'eg: MyAwesomeChannel',
```
Comment (HenryHengZJ) [inline packages/components/nodes/Telegram/Telegram.ts:44]: same here, default as empty and set it to `placeholder`

File: packages/components/package.json [hunk 3]
```diff
@@ -1,49 +1,49 @@
...
-    "prettier": "^2.7.1",
-    "typescript": "^4.6.3"
-  }
+    "name": "outerbridge-components",
+    "version": "1.0.4",
+    "description": "Outerbridge Components",
+    "main": "dist/src/index",
+    "types": "dist/src/index.d.ts",
+    "scripts": {
+        "build": "rm -rf dist && tsc && gulp",
```
[the remark this answers is not in the document]
Comment (tylim88) [reply packages/components/package.json:8]: I am using window and bash, this command can run properly

[the remark this answers is not in the document]
Comment (tylim88) [reply packages/components/package.json:8]: I add clean to build then

[the remark this answers is not in the document]
Comment (HenryHengZJ) [reply packages/components/package.json]: Git bash works but windows cmd doesnt support rm -rf

File: packages/ui/src/ui-component/InputParametersTooltip.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import { Info } from '@mui/icons-material';
+import { IconButton, Tooltip } from '@mui/material';
+import parse from 'html-react-parser';
+
+export const InputParameterTooltip = ({ title }) => {
```
Comment (HenryHengZJ) [inline packages/ui/src/ui-component/InputParametersTooltip.js:5]: can we pull from latest master and replace all occurrence of `Tooltip` to use this instead? Going to decline #82 and have the bug #81 fixed as part of this PR

File: packages/ui/package.json [hunk 5]
```diff
@@ -50,7 +50,7 @@
     "scripts": {
         "start": "react-scripts start",
         "dev": "react-scripts start",
-        "build": "react-scripts build",
+        "build": "env CI=false react-scripts build",
```
Comment (HenryHengZJ) [inline packages/ui/package.json:53]: when I yarn build locally `env env' is not recognized as an internal or external command,
operable program or batch file.`

Comment (tylim88) [reply packages/ui/package.json]: try the latest commit

File: packages/ui/src/ui-component/TooltipWithParser.js [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+
+export const TooltipWithParser = ({ title }) => {
+    return (
+        <Tooltip title={parser(title)} placement="right">
+            <IconButton>
+                <Info style={{ height: 18, width: 18 }} />
+            </IconButton>
+        </Tooltip>
+    );
+};
```
Comment (HenryHengZJ) [inline packages/ui/src/ui-component/TooltipWithParser.js:13]: Warning `'title' is missing in props validation  react/prop-types`
Need to `import PropTypes from 'prop-types';` and use it
`

File: packages/ui/package.json [hunk 7]
```diff
@@ -50,7 +50,7 @@
     "scripts": {
         "start": "react-scripts start",
         "dev": "react-scripts start",
-        "build": "react-scripts build",
+        "build": "cross-env CI=false react-scripts build",
```
Comment (HenryHengZJ) [inline packages/ui/package.json:53]: what's the purpose of `cross-env CI=false`?

Comment (tylim88) [reply packages/ui/package.json:53]: if CI=true, reacr-script turn eslint warning into error, failing compilation

in CI environment, CI is set to true

Comment (tylim88) [reply packages/ui/package.json:53]: https://stackoverflow.com/questions/64518226/my-create-react-app-is-failing-to-compile-due-to-eslint-error

File: packages/ui/src/ui-component/TooltipWithParser.js (hunk 6 above)
Comment (tylim88) [reply packages/ui/src/ui-component/TooltipWithParser.js:14]: resolved

**Review by HenryHengZJ — APPROVED** [review]

File: packages/ui/package.json (hunk 7 above)
Comment (tylim88) [reply packages/ui/package.json:53]: another detail
https://stackoverflow.com/questions/50930044/what-does-export-ci-true-do-in-this-create-react-app-deploy-script

**Pull request merged**